src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -693,6 +700,19 @@ export default Vue.extend({
       }
     },
 
+    mouseScrollSkip: function (event) {
+      if (event.target && (event.target.matches('.vjs-tech') || event.target.matches('.ftVideoPlayer'))) {
```

Comment:
Maybe add some code comment to explain the selectors?

Reply:
To be honest I have no idea why, but targeting just 'vjs-tech' doesn't seem to work all the time. 99% of the time it works just fine  but in some rare cases '.ftVideoPlayer'  gets targeted on hover and I don't know why. I added it as a selector just to account for that rare case. If I can figure out what's causing it maybe I can remove it.